eng/Versions.props

```diff
@@ -4,7 +4,10 @@
 
   <PropertyGroup>
     <VersionSDKMinor>1</VersionSDKMinor>
-    <VersionPrefix>11.0.$(VersionSDKMinor)00</VersionPrefix>
+    <MajorVersion>11</MajorVersion>
+    <MinorVersion>0</MinorVersion>
+    <MajorMinorVersion>$(MajorVersion).$(MinorVersion)</MajorMinorVersion>
```

Comment:
I would prefer to avoid this MajorMinorVersion property and just concatenate in the places that need it, honestly.

Reply:
Fixed with https://github.com/dotnet/dotnet/pull/3895/commits/d908f16e87ef1e358ccfcd5656799b286078038a